tests/framework/installer/nfs_manifests.go

```diff
@@ -30,14 +30,156 @@ kind: CustomResourceDefinition
 metadata:
   name: nfsservers.nfs.rook.io
 spec:
+  additionalPrinterColumns:
```

Comment:
This is another location where we are defining a CRD, so we should make sure it's consistent with the other locations we define it too.  This is just for tests, but it's good to be consistent all around.

I make this comment specifically thinking about the suggestion I gave to add the AGE printer column :)

Reply:
addressed https://github.com/rook/rook/pull/5650#discussion_r446629196